core-modular/dev-tools/templates/simple/luigi-ui/lui-ui5wc.js

```diff
@@ -573,8 +573,110 @@ const connector = {
     dialog.appendChild(ui5Toolbar);
     document.body.appendChild(dialog);
     dialog.open = true;
+  },
+
+  openUserSettings: (settings) => {
+    if (!settings) {
+      settings = {
+        title: 'User Settings'
+      };
+    }
+
+    const processUserSettingGroups = () => {
+      const config = window.Luigi?.getConfig();
```

Comment:
I think preparing/reading the config data should be done in core-headless. In addition if there are coming from a user settings service it will not work.
In addition ui5 webcomponent offers a [user settings dialog](https://sap.github.io/ui5-webcomponents/components/fiori/UserSettingsDialog/).
Let's clarify in the daily